war/src/main/resources/images/symbols/rss.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="512px" height="512px" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g transform="translate(81.000000, 81.323000)" fill-rule="nonzero" stroke="currentColor" stroke-width="32">
+            <path d="M28.552,293.458 L28.2086637,293.462282 C12.7932399,293.768174 0.411027957,306.374012 0.439973838,321.849687 C0.469416663,337.481891 13.1509336,350.138264 28.7831021,350.136598 C44.2589396,350.13495 56.8401627,337.727807 57.1154778,322.311817 L57.119,321.927 L57.1149621,321.530547 C56.8237248,306.313721 44.6521378,293.97949 29.47116,293.473221 L29.045,293.462 L28.552,293.458 Z" fill="currentColor"></path>
```

Comment:
This SVG could be optimized, I'd suggest using https://jakearchibald.github.io/svgomg/

Reply:
I ran it through that service but it seemingly broke the SVG? :P I've updated the icon to look a little nicer.